### DISCUSSION TITLE
Lost SRM

### DISCUSSION THREAD

**Discussion by adroFPV** [wormhole-foundation/wormhole#338] [Problems / bugs] [answered]
Hello, i'm wondering if someone can help, i've transferred SRM through the WormholeBridge and now it's lost
It went through from ETH to SOL but it was some unrecognisable wrapped token, i couldn't find any dex that would recognise it, so i sent it back through the wormhole and now it doesn't appear in SOL or ETH wallet

Transactions: 

2ddGDfMhKhViqX5C2xcJyLr14XZ48dCRrZHaf5mx14RHaXKuVbriC8CBeiqXPak3gRnbgektQipypHwnVFgaMkE2
Block: #92383097

4McJgq9vwRcSbPbCH1WZfYxL9Xc1pAtHjWZjKVgdwEBPyWk3hvwKh8UaSopD9YLh1tqgf9UaT9p1bAPSfU7S2VuT
Block:  #92383757

65kzfN9bAQL1Z1WRjKAvY8gAedL98sKrPxvJrthkVVEHzocwMCG1QkbVKkdBTveN6BshRxtK6zoS2Q545omtyiL6
Block: #92415177

5bz1i5X9wy5SFsubnB1ZdXeQcHqUqhwyxiJ5CbgCoRAXGvK8rCdmvuNPDS8t3RWLQ3dVG63gurqd9dL4m4ZZJ9a9
Block: #92493045

 2EoVndjdJwFWzqk5YfAkJm4jpxdvSSpZ1a4PeKyjMyDagTopaegcHtvzXuKokppnYhJW7BUW4QogzK3wdyeKmkgB
Block: #92495849

Hope this helps. Let me know if you need anything else. 

Thanks,
Adrian

**Comment by leoluk** [comment, marked as answer]
Looks like this is the correct transaction: https://explorer.solana.com/tx/65kzfN9bAQL1Z1WRjKAvY8gAedL98sKrPxvJrthkVVEHzocwMCG1QkbVKkdBTveN6BshRxtK6zoS2Q545omtyiL6

The transaction was successfully processed by the Wormhole network, but the wallet you're using may not have properly sent the Eth transaction. Try clearing your cache and reloading wormholebridge.com to see if the transaction can be completed there. If you need help with that, contact the wormholebridge.com team: https://github.com/solana-labs/oyster/issues

If wormholebridge.com doesn't work, you can manually submit the signed VAA bytes below to the submitVAA method on the Wormhole contract: https://etherscan.io/address/0xf92cd566ea4864356c5491c177a430c222d7e678#writeContract

---

Digest: 1dc1c64cb4abb3b9f853f5ca40036fee5889f43e746200ca1f02cf41ee29b1d5

Signed VAA bytes:

> 01000000010d006d24b186ee57b659dc03feb4714c65db8d292cdc1fccaf07f41f968fc988756834efc547fde32359626a5d7d42004488d58e74b32fedf2c9f06b9c5682c69d0501019fe769f059c0f035e8f7cf1387dc233eb165d6699a86aa883c52cdd14d5f46f27dda59d8c4fa7d1c7a9878fa37524efd80d6e4cd2a1163a82aa24cbe5c4db1250102bdd866551d694f646e8e1be190da28d807d808e30c1d575cde9488fae0a183a70f1cceb1e8b334fd1aafdde0dee24bc2ac0e819f060fcc1772fecf39cd32e99001048f75e5ace70a04b79163a7a6d7fcb891a044118a8a2d320fd5f7c23572ad35b869947e5128ba2999a75788ce7c6e158512e8ca631de2b30aa067fea7de69fa950105dea7f1da9629b329cd7e402ba307656adf3cb4d577feb9fe37ecb4fa355e4c776785a0dbf45c747258fe9cd8f149a9c76562b6ac16d27de5de5eaff5f45766f1000764ad5afba5dcf860c587f666a1efeae1973488eb1680f60c6a74d9ee9398e869123f111e89f956fc065e43c16a0ce8eb3a9e7d715980b9133b157141acded02101083557df6de54f5225a650a5d1e753910b4946251938c40d71bb3225ec833f3dd10e1aea4971a923425183fd8446bf691e8bc7b28a8dc4ee2432e250a15ba29080010a3b0719a6cdf4b587d7333ce833f4847463658c15fc94fcc4240dece99899680628fffbf0681619e4473e92686b4815733413962844706fa620aef9722447eff0000bed7f55458ac2ab5db69b3cd285541ac2fcd3b2c4f226c14d5cf3da5283eb0ce81c2efbd8c06042488845fd929e1a73b6605829aad9d557aee444311f3c0b0441000da02dbd25b02c5ff337aa0ae9def6aec7a75568a71dd59103b938e67a8a8ba285200df816fbb3a58b6f5c9da046403741370b26c6c71868e8346ba3c0cd5cb30c0110900ddc603284577404e10a2872105b8551e067b957a4ec826718dc096604fb52781f999f33d59e7d6f44e095015f57b245b8efb2995a4c0af2517912fea7726c0011621ab712cfdd8a5dc35a0c112803e0232b176798e3f70cfac573cf940024ec4b54b7bcb7192e8fd1bc3605c54fa5de68080369351a657c31aefef817356255f901122f6348e331559d8e397484333cc52f1dd2635481fe66d801985779a3be7de33c32207342858f3e812f193232c03fda6796113b3694a6ad95a02c8888f02e117500611eaf4e100000567801025517bb7f5b0f7b2ead03649dd94a14771d928ea236acf57f69596e367542f23c000000000000000000000000aa4ea2c84037c2085459081400f52cd92e30f3bb02000000000000000000000000476c5e26a75bd202a9683ffd34359c0cc15be0ff060000000000000000000000000000000000000000000000000000000005e73219

**Reply by adroFPV** [reply]
I've cleared my cache so i'm logged out of sollet wallet and now i cant find my seed phrase. Is there anyway i can revert this using archived cache or something?

**Reply by leoluk** [reply]
You definitely need the seed phrase of the wallet if you send the tokens back to Solana.

**Comment by adroFPV** [comment]
Ahh ok, my bad... I've run it again and gas fees coming in at $150 so i might try again in the morning if its a little more reasonable. Thanks for your help so far.
